src/ListingPage.php

```diff
@@ -192,4 +188,53 @@ public function view(): void
         do_action('enqueue_google_tag_manager_script', $this->context);
         Timber::render($this->templates, $this->context);
     }
+
+    public static function get_sticky_posts(): array
```

Comment:
This `static` function helps to override the main query requested from `functions.php`